vendor/bat-native-ads/src/bat/ads/internal/ads/new_tab_page_ads/new_tab_page_ad_unittest.cc

```diff
@@ -193,26 +184,8 @@ TEST_F(BatAdsNewTabPageAdTest, DoNotFireEventWithInvalidCreativeInstanceId) {
   ExpectAdEventCountEquals(ConfirmationType::kViewed, 0);
 }
 
-TEST_F(BatAdsNewTabPageAdTest, DoNotFireEventWhenNotPermitted) {
-  // Arrange
-  const CreativeNewTabPageAdInfo& creative_ad = BuildAndSaveCreativeAd();
-
-  // Act
-  new_tab_page_ad_->FireEvent(kUuid, creative_ad.creative_instance_id,
-                              mojom::NewTabPageAdEventType::kViewed);
-
-  // Assert
-  EXPECT_FALSE(did_serve_ad_);
-  EXPECT_FALSE(did_view_ad_);
-  EXPECT_FALSE(did_click_ad_);
-  EXPECT_TRUE(did_fail_to_fire_event_);
-
-  ExpectAdEventCountEquals(ConfirmationType::kViewed, 0);
-}
-
 TEST_F(BatAdsNewTabPageAdTest, DoNotFireEventIfCreativeInstanceIdWasNotFound) {
   // Arrange
-  new_tab_page_ads::frequency_capping::ForcePermissionRules();
```

Comment:
See previous comment concerning `This logic is still required if ads are disabled`